sql/driver/driver_test.go

```diff
@@ -720,113 +719,3 @@ func TestSelectNoTable(t *testing.T) {
 		}
 	}
 }
-
-func TestInsecure(t *testing.T) {
```

Comment:
TestInsecure and TestPrivileges were moved verbatim to permissions_test (TestPrivileges was renamed TestGrants)